docusaurus/docs/shared/create-plugin-backend.md

```diff
@@ -0,0 +1,45 @@
+The Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin) is a CLI application that simplifies Grafana plugin development, so that you can focus on code. The tool scaffolds a starter plugin, all the required configuration, and a development environment using [Docker Compose](https://docs.docker.com/compose/) for you.
+
+1. <span>In a new directory, create a plugin from a template using the create-plugin tool. When prompted for the kind of plugin, select {props.pluginType}</span> and answer yes to "Do you want a backend part of your plugin?":
```

Comment:
Not super important, but it's not a best practice to include HTML elements in markdown. Why are these span tags here? (this question also for subsequent span tags)

Reply:
All `<span>` tags I've added are so that the `{props.pluginType}` correctly updates to i.e. `datasource` when the shared component is referenced like `<CreatePlugin pluginType="datasource" />`. Without them the page just shows `{props.pluginType}`. If there's an alternative I'm unaware of - happy to update.